README.md

```diff
@@ -1 +0,0 @@
-# credit_card_validator
```

Comment:
Readme always must be filled.

It is a face of your application! Here you should have good description what are you doing exactly, how to run app, what it contains and so on.